src/Client/HedgedConnections.cpp

```diff
@@ -278,10 +283,11 @@ Packet HedgedConnections::drain()
 Packet HedgedConnections::receivePacket()
 {
     std::lock_guard lock(cancel_mutex);
-    return receivePacketUnlocked({});
+    return receivePacketUnlocked({}, 0);
 }
 
-Packet HedgedConnections::receivePacketUnlocked(AsyncCallback async_callback)
+/// TODO(ab): make draining_timeout_seconds work for hedged connection.
```

Comment:
This TODO is important. I suppose we want to enable `use_hedged_requests` by default soon ...